core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -900,7 +881,6 @@ object DynamicListenerConfig {
    */
   val ReconfigurableConfigs = Set(
     // Listener configs
-    SocketServerConfigs.ADVERTISED_LISTENERS_CONFIG,
```

Comment:
@FrankYang0529 Could you please file a minor follow-up to add this change to `zk2kraft.html`?

Reply:
Yes, I will create a minor PR for it. Thanks.